### ISSUE TITLE
Unable to use with Anaconda python distribution

### ISSUE THREAD

**Issue by alanhamlett** [wakatime/vscode-wakatime#28]
Getting this error when running wakatime cli:

```
wakatime: error: unrecognized arguments: vscode-wakatime/1.1.18
```

Python: `Python 2.7.5 :: Anaconda 1.8.0 (x86_64)`
OS: Mac OS X

The plugin argument contains a space and is being parsed as two separate arguments.

**Comment by alanhamlett** [comment]
Fixed in v1.1.19.

**Comment by phaberest** [comment]
I'm afraid I'm getting the same error on 1.2.0

`wakatime: error: unrecognized arguments: vscode-wakatime/1.2.0`
